packages/dd-trace/src/config.js

```diff
@@ -200,8 +200,11 @@ class Config {
       process.env.DD_TRACE_TELEMETRY_ENABLED,
       !process.env.AWS_LAMBDA_FUNCTION_NAME
     )
-    const DD_TELEMETRY_DEBUG_ENABLED = coalesce(
-      process.env.DD_TELEMETRY_DEBUG_ENABLED,
+    const DD_TELEMETRY_HEARTBEAT_INTERVAL = process.env.DD_TELEMETRY_HEARTBEAT_INTERVAL
+      ? parseInt(process.env.DD_TELEMETRY_HEARTBEAT_INTERVAL) * 1000
+      : 60000
+    const DD_TELEMETRY_DEBUG = coalesce(
```

Comment:
Why not keep both names for backward compatibility?

Reply:
It is an IAST property and nobody is using it yet

Reply:
Is this even used anywhere at this point?